src/components/SideMenu.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, IconButton, styled } from '@mui/material';
+import { ReactElement, ReactNode } from 'react';
+
+interface SideMenuProps {
+  children: ReactNode;
+  expanded?: boolean;
+  minimizedMenu?: ReactElement;
+}
+
+export const StyledMinimizedMenuButton = styled(IconButton)(({ theme }) => ({
+  background: theme.palette.primary.main,
+  ':hover': { background: theme.palette.primary.dark },
+  color: theme.palette.common.white,
+  borderRadius: '0',
+}));
+
+export const SideMenu = ({ children, expanded, minimizedMenu }: SideMenuProps) => (
```

Comment:
dårlig visning på mobil her, men lurer på om vi bare skal utsette retting av det akkurat nå for å komme i mål med v1

Reply:
har rettet opp i noen småting slik at det ser litt bedre ut på mobil også nå. Men fortsatt en lang vei å gå